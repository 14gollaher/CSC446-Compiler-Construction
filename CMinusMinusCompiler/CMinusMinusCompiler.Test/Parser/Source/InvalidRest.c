int main()
{
	int x%;
	return 4.2;
}