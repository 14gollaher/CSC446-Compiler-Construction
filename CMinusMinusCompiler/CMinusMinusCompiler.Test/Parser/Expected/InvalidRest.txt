ERROR: Line 3 Expected token "SemiColonToken" - Received token "MultiplicationOperatorToken"
ERROR: Line 4 Expected token "RightBraceToken" - Received token "SemiColonToken"
ERROR: Line 4 Unexpected tokens in source file, expected End-of-File Token
Completed processing InvalidRest.c
